remote-query/remote-query-client/src/main/resources/org/infinispan/query/remote/client/query.proto

```diff
@@ -2,8 +2,10 @@ import "org/infinispan/protostream/message-wrapping.proto";
 
 package org.infinispan.query.remote.client;
 
+// Allocated type ids are: 4400 -> 4599 (see org.infinispan.commons.marshall.ProtoStreamTypeIds)
```

Comment:
We should probably add these to ProtoStreamTypeIds  to prevent accidental duplicates

Reply:
Right!